src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27515.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue27515 : _IssuesUITest
+    {
+        public Issue27515(TestDevice testDevice) : base(testDevice)
+        {
+        }
+
+        public override string Issue => "Slider's thumb image doesn't respect color";
+
+        [Test]
+        [Category(UITestCategories.Slider)]
+        public void SliderThumbImageShouldRespectColor()
+        {
+            App.WaitForElement("ChangeThumbImageButton");
+            App.Click("ChangeThumbImageButton");
+            App.Click("ChangeThumbColorButton");
+            VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build:
![Uploading image.png…]()